lib/radiator/directory/editor/manager.ex

```diff
@@ -84,9 +84,19 @@ defmodule Radiator.Directory.Editor.Manager do
   end
 
   def create_audio(network = %Network{}, attrs) do
-    Ecto.build_assoc(network, :audios)
-    |> Audio.changeset(attrs)
-    |> Repo.insert()
+    Multi.new()
```

Comment:
wondering if we should change that to be `create_audio_publication` instead now. And one only can create an episode or an audio_publication and the audio is implied.

Reply:
I'm doing #178 next and will keep this in mind. 

We could `create_audio/2` in Podcast/Network with Episode/AudioPublication as side-product to keep it consistent.

Or get rid of `create_audio` entirely and let `create_episode`/`create_audio_publication` create the audio?

The API workflow in either case would be to create an audio immediately in one of both publication contexts.